dev/ci/appveyor.bat

```diff
@@ -37,5 +38,5 @@ if %USEOPAM% == true (
 GOTO :EOF
 
 :ErrorExit
-  ECHO ERROR MakeCoq_MinGW.bat failed
+  ECHO ERROR appveyor.bat failed
```

Comment:
I believe you can use `%0` to get the name of the batch file, and will still work if the file name changes.